codegen/runner/runner.go

```diff
@@ -38,11 +38,11 @@ type stackTracer interface {
 
 func checkError(err error, message string) {
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "%s:\n%s\n", message, err)
+		fmt.Printf("%s:\n%s\n", message, err)
```

Comment:
I know you did this to pass linter, but what you need is on error print it to stderr not stdout